tests/integration/test_map_reduce.py

```diff
@@ -13,24 +13,25 @@
 # limitations under the License.
 #
 
-import random
-
 import numpy as np
+import pytest
 
 import cunumeric as num
 
 
-def test():
-    anp = np.random.randn(4, 5)
-    b = random.randint(1, 13)
-    a = num.array(anp)
+def test_basic():
```

Comment:
I'm very confused by the changes in this file. Feels like they are from a totally different file. (the file names above are equally confusing.) can you double check if this is truly due to the incorrect matching of files?

Reply:
I think maybe there was an intermediate copy-pasta? But looking at the actual current test on `branch-22.05` right now, I think the end result is correct? 

https://github.com/nv-legate/cunumeric/blob/branch-22.05/tests/map_reduce.py

Please confirm

Reply:
looks good.